services/movie.py

```diff
@@ -0,0 +1,35 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[id] = None
+) -> QuerySet:
```

Comment:
You could indicate that you return a `QuerySet` of `Movie` objects in the return type annotation. Revise your type annotation across methods to encounter this.